src/tests/shipments_api.test.ts

```diff
@@ -70,6 +76,36 @@ describe('Shipments API', () => {
         },
       })
 
+      expect(res.errors).not.toBeUndefined()
+      expect(res.errors).not.toBeEmpty()
+
+      if (res.errors == null || res.errors.length === 0) {
+        return
+      }
+
+      expect(res.errors[0].message).toEqual(
+        'addShipment forbidden to non-admin users',
+      )
```

Comment:
Can you use optional chaining here? Might allow you to get rid of the check on line 82:

```typescript
expect(res?.errors?.[0]?.message).toEqual(
  'addShipment forbidden to non-admin users',
)
```

Reply:
It does! Thank you!